java/core/src/java/org/apache/orc/impl/InStream.java

```diff
@@ -87,6 +87,11 @@ protected void reset(DiskRangeList input) {
     setCurrent(input, true);
   }
 
+  protected void reset(DiskRangeList input, long length) {
```

Comment:
Please add a function description for this method because this method will be used as a wrapper from now.

Reply:
Thanks for your comment, done.